DependencyInjection/Configuration.php

```diff
@@ -221,6 +221,7 @@ private function getDbalConnectionsNode(): ArrayNodeDefinition
                 ->end()
                 ->booleanNode('disable_type_comments')->end()
                 ->scalarNode('server_version')->end()
+                ->integerNode('idle_connection_ttl')->defaultValue(30)->end()
```

Comment:
let's discuss the default value: MySQL defaults to 8h, other DB engines don't configure this AFAIK
let's raise this to e.g 1h? any other proposals (and reasoning)?